### PR TITLE
Add ability to set Airbrake host name.

### PR DESCRIPTION
**Pull request by vmihailenco**

### CODE REVIEW

File: notifier.go [hunk 1]
```diff
@@ -15,37 +15,54 @@ import (
...
+}
+
 type filter func(*Notice) *Notice
 
 var httpClient = &http.Client{
 	Transport: &http.Transport{
 		Proxy: http.ProxyFromEnvironment,
 		Dial: (&net.Dialer{
-			Timeout:   30 * time.Second,
+			Timeout:   15 * time.Second,
```
Comment (kyrylo) [inline notifier.go]: Seems to be irrelevant to this PR (icluding line 40).

File: notifier.go [hunk 2]
```diff
@@ -15,37 +15,54 @@ import (
...
 			KeepAlive: 30 * time.Second,
 		}).Dial,
 		TLSHandshakeTimeout: 10 * time.Second,
 		TLSClientConfig: &tls.Config{
 			ClientSessionCache: tls.NewLRUClientSessionCache(1024),
 		},
-		MaxIdleConnsPerHost:   100,
+		MaxIdleConnsPerHost:   10,
```
Comment (kyrylo) [inline notifier.go]: Could you please explain why this needs tweaking?

Comment (vmihailenco) [inline notifier.go]: There are no good reasons except that it looks safer to me. Feel free to propose different number if you know how to configure it properly.
